src/main/scala/io/citrine/lolo/transformers/Standardizer.scala

```diff
@@ -83,6 +92,19 @@ class StandardizerTrainingResult(
   override def getModel(): Model[PredictionResult[Any]] = new StandardizerModel(baseTrainingResult.getModel(), outputTrans, inputTrans)
 
   override def getFeatureImportance(): Option[Vector[Double]] = baseTrainingResult.getFeatureImportance()
+
+  override def getPredictedVsActual(): Option[Seq[(Vector[Any], Any, Any)]] = {
+    baseTrainingResult.getPredictedVsActual() match {
+      case None => None
+      case Some(predictedVsActual) => Some(
+        (
+          Standardizer.invertStandardization(predictedVsActual.map(_._1), inputTrans),
+          Standardizer.invertStandardization(predictedVsActual.map(_._2), outputTrans),
+          Standardizer.invertStandardization(predictedVsActual.map(_._3), outputTrans),
+        ).zipped.toSeq
```

Comment:
I get the same deprecation warning here for `.zipped`

Reply:
We can't use `lazyZip` because we compile with Scala 2.12 in addition to 2.13

Reply:
Not really related to this PR but... do you know how long we can expect to maintain that?

Reply:
That's a good question. I think that we made Lolo cross-compatible to facilitate the Mithril migration. Now that that's done, perhaps we can bump Lolo to 2.13.
On the other hand, since it is a public library, there might be value in maintaining backwards compatibility for a while.